tests/resources/telescope_positions-North-utm.ecsv

```diff
@@ -7,17 +7,27 @@
 # - {name: utm_north, unit: m, datatype: float32}
 # - {name: alt, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
 # - {name: geo_code, datatype: string, description: Geographical Position Code}
-# meta: !!omap
-# - data_type: positionfile
-# - array_name: "North-ArrayElements"
-# - description: "CTAO Northern Array Element Coordinates"
-# - center_lon: "-17.8920302 deg"
-# - center_lat: "28.7621661 deg"
-# - center_alt: "2177m"
-# - center_northing: "3185066.278 m"
-# - center_easting: "217611.227 m"
-# - corsika_obs_level: "2158 m"
-# - EPSG: 32628
+# meta:
+#   EPSG: 32719
+#   center_alt: !astropy.units.Quantity
+#     unit: &id001 !astropy.units.Unit {unit: m}
+#     value: 2150.0
+#   center_easting: !astropy.units.Quantity
+#     unit: *id001
+#     value: 217654.06411838497
+#   center_lat: !astropy.units.Quantity
+#     unit: &id002 !astropy.units.Unit {unit: deg}
+#     value: 28.762158
+#   center_lon: !astropy.units.Quantity
+#     unit: *id002
+#     value: -17.891571
+#   center_northing: !astropy.units.Quantity
+#     unit: *id001
+#     value: 3185065.2899643113
```

Comment:
Why was this changed from the original?

Reply:
same as above

Reply:
Could we agree on reasonable precision? It seems we are in the micrometer range here?

Reply:
Sure. Orel pointed me to the values we define in the CTA gitlab. I will make sure that all the files here have exactly those values. would that be a good solution?